torch/csrc/jit/import.cpp

```diff
@@ -498,37 +482,11 @@ void buildBlock(const Graph_& graph_, Block* block,
     block->appendNode(node);
   }
 
-  for (auto output : graph_.outputs) {
-    if (value_map.find(output.name) != value_map.end()) {
-      Value* v = value_map.at(output.name);
-      block->registerOutput(v);
-    }
-  }
-}
-
-void buildBlocks(const std::vector<Graph_>& graphs_, Node* node,
-                 std::unordered_map<std::string, Value*>& value_map) {
-  for (auto g_ : graphs_) {
-    auto block = node->addBlock();
-    buildBlock(g_, block, value_map);
-  }
-}
-
-std::shared_ptr<Graph> buildGraph(const Graph_& graph_) {
-  auto graph = std::make_shared<Graph>();
-  std::unordered_map<std::string, Value*> value_map;
-
-  buildBlock(graph_, graph->block(), value_map);
-
-  return graph;
-}
-
-std::vector<std::shared_ptr<Graph>> buildGraphs(const std::vector<Graph_>& graphs_) {
-  std::vector<std::shared_ptr<Graph>> graphs;
-  for (auto g_ : graphs_) {
-    graphs.push_back(buildGraph(g_));
+  for (auto & output : graph_.outputs) {
+    assert(value_map.find(output.name) != value_map.end());
```

Comment:
This assert is not necessary. We usually compile with `-DNDEBUG` so it won't even be there, but `.at` already does the error checking for you.